### PR TITLE
Feat/add patch to cors

### PR DESCRIPTION
**Pull request by phaelar**
adding PATCH as a default allowable method for the CORS middleware

### CODE REVIEW

File: packages/server-boilerplate-middleware/README.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ Defaults to:
 ```js
 const corsOptions = {
   allowedHeaders: [],
-  allowedMethods: ['GET', 'POST', 'PUT', 'DELETE', 'OPTIONS'],
+  allowedMethods: ['GET', 'POST', 'PATCH', 'PUT', 'DELETE', 'OPTIONS'],
```
Comment (leokleok) [inline packages/server-boilerplate-middleware/README.md]: can you update the allowed Methods in  `line 158` as well on this readme?

File: packages/server-boilerplate-middleware/README.md [hunk 2]
```diff
@@ -232,27 +232,27 @@ lerna bootstrap;
 ```
 
 ### Running Tests
-To run the tests during development, use:
+To run the tests during development, use at the root directory:
 
 ```sh
-npm run test:watch;
+npx lerna run --scope @mcf/server-boilerplate-middleware test:watch
```
Comment (leokleok) [inline packages/server-boilerplate-middleware/README.md]: 👍  thanks fo updating

File: packages/server-boilerplate-middleware/src/security/cors.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ const DEFAULT_ALLOWED_ORIGINS: string[] = [];
 const DEFAULT_CREDENTIALS = true;
 const DEFAULT_PREFLIGHT_CONTINUE = false;
 
-export type HttpMethod = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'OPTIONS';
+export type HttpMethod = 'GET' | 'POST' | 'PATCH' | 'PUT' | 'DELETE' | 'OPTIONS';
```
[the remark this answers is not in the document]
Comment (phaelar) [reply packages/server-boilerplate-middleware/src/security/cors.ts]: oops, forgot to put that in after reverting the changes

File: packages/server-boilerplate-middleware/README.md (hunk 1 above)
Comment (phaelar) [reply packages/server-boilerplate-middleware/README.md]: 👌

**Pull request merged**